include/alpaka/acc/AccGenericSycl.hpp

```diff
@@ -23,6 +23,7 @@
 #    include <alpaka/intrinsic/IntrinsicGenericSycl.hpp>
 #    include <alpaka/math/MathGenericSycl.hpp>
 #    include <alpaka/mem/fence/MemFenceGenericSycl.hpp>
+#    include <alpaka/meta/SyclPrintf.hpp>
```

Comment:
That seemed like a good place to put the redefinition of `printf` for all SYCL files without going one by one

Reply:
The idea behind `SyclPrintf.hpp` is to be able to use `printf` inside SYCL kernels (through `sycl::ext::oneapi::experimental::printf`) instead of `sycl::stream`. If you prefer we can either include that file directly in `alpaka.hpp` or put the content of the file directly at the beginning of `AccGenericSycl.hpp` (similar to what has been done in `AccOmp5.hpp` to avoid `printf` for AMD GPU)

Reply:
Okay, that makes sense.